kit/scrolls/src/Lists/UIAccordionSectionList/UIAccordionSectionList.tsx

```diff
@@ -222,9 +222,21 @@ function UIAccordionSectionListInner<ItemT>({
     const [foldedSections, setFoldedSections] = React.useState<Record<string, boolean>>({});
     // This is used to reduce re-creation of a press callback
     const foldedSectionsHolderRef = React.useRef<Record<string, boolean>>(foldedSections);
+
     React.useEffect(() => {
+        // Add initial isFolded value of the corresponding section to the foldedSections list
+        sections.forEach(section => {
+            const { key, isFolded } = section;
+            const sectionKey = `${key}:header`;
+            if (isFolded && !(sectionKey in foldedSections)) {
+                setFoldedSections({
+                    ...foldedSections,
+                    [sectionKey]: true,
+                });
+            }
+        });
```

Comment:
Can you please explain how exactly does it fix the problem? 🤔 
I see that there is already a logic for that https://github.com/tonlabs/UIKit/pull/1613/files#diff-9603277eb9363ebb5ad83739f59f6641f63233d3311873559ee81b916ac2a902R244-R252
how does it differ from that?

Reply:
It fixes the initial state of foldedSections - we start mentioning isFolded prop only after first click on initially folded section.

Reply:
Ok, I see few problems here:
1. It triggers n re-renders (initial render, and then n re-renders based on amount of `setFoldedSections` calls)
2. It's going to run on every sections change (not only once after mount)
3. It's going to run the second time, after you change `foldedSections` (second argument to useEffect will be changed) (even though it won't call `setFoldedSection`, so it's a just waste of CPU resources)
4. The logic I [mentioned](https://github.com/tonlabs/UIKit/pull/1613/files#diff-9603277eb9363ebb5ad83739f59f6641f63233d3311873559ee81b916ac2a902R244-R252), that is actually about the same, either one of them must be deleted